sonar-scanner-engine/src/main/java/org/sonar/batch/issue/TrackedIssueAdapter.java

```diff
@@ -92,9 +93,14 @@ public String resolution() {
     return issue.resolution();
   }
 
+  /**
+   * @deprecated since 5.5, manual issue feature has been dropped.
+   */
+  @Deprecated
+  @CheckForNull
   @Override
   public String reporter() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)